New layout and sections (#51)
dandi/handbook: docs/10_using_dandi.md

# Using DANDI

The DANDI project can be represented schematically by the figure:

<img
src="./img/dandi_structure.svg"
alt="dandi_structure"
style="width: 70%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>

> The Clients are DANDI Python CLI and DANDI Web application and the Server side contains a RESTful API and DANDI JupyterHub. The file organization to store data together with metadata is called `Dandiset`.

Reviewer: "The Client side `contains"```

> The Client Site contains DANDI Python CLI and DANDI web application, the Server site contains RESTfulAPI and DANDI JupyterHub. The basic structure to store data together with metadata is called `Dandiset`.

Reviewer 2: ```suggestion
The Clients are DANDI Python CLI and DANDI Web application and the Server side contains a RESTful API and DANDI JupyterHub. The file organization to store data together with metadata is called `Dandiset`.
```

<img
src="./img/dandiset_activity.svg"
alt="dandiset_activity"
style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>

You can learn more about the `Dandiset` acctions in separate sections:

- [View](./11_view.md)
- [Download](./12_download.md)
- [Upload](./13_upload.md)
- [Publish](./14_publish.md)

## DANDI Components

### **The DANDI Web application**

The [DANDI Web application](https://dandiarchive.org/) allows you to:

* Search across all public `Dandisets`
* Download data from public `Dandisets`
* Create a new `Dandiset` and provide metadata
* Publish your `Dandiset`

### **The DANDI Python client**

The [DANDI Python client](https://pypi.org/project/dandi/) allows you to:

* Download `Danidsets` and individual subject folders or files
* Organize your data locally before upload
* Upload `Dandisets`

Before you can use the DANDI Python client, you have to install the package with `pip install dandi` in a Python 3.7+
environment.

You should check the [Dandi Debugging section](./15_debugging.md) in case of any problems.

### **The Dandihub analysis platform**

[Dandihub](https://hub.dandiarchive.org) provides:

> - a JupyterHub instance in the cloud to interact with the data stored in DANDI.

Reviewer: I would make this one sentence versus having a single bullet item

Anyone on the Internet can view and download public `Dandisets`. Registered users can also create `Dandisets`, upload data, and publish the `Dandiset` to generate a DOI for it.

<img

> src="./img/dandiset_activity.svg"

Reviewer 2: ```suggestion
src="./img/dandiset_activity.png"
```

Reviewer 2: svg is fine.
